lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnDriverConfigurationProviderImpl.java

```diff
@@ -33,7 +33,8 @@
 /**
  * Default driver configuration provider for yarn runtime.
  */
-final class YarnDriverConfigurationProviderImpl implements DriverConfigurationProvider {
+public final class YarnDriverConfigurationProviderImpl implements DriverConfigurationProvider {
```

Comment:
We need it public so it can be referenced from `UnmanagedAmYarnClientConfiguration` which is in another package (`unmanaged`).